xml/ay_custom_user_scripts.xml

```diff
@@ -430,7 +433,9 @@ echo "Testing the pre script" &gt; /tmp/pre-script_out.txt
          <para>
           Optional; default is the type of the script (pre-scripts in this
           case). If you have more than one script, you should define
-          different names for each script.
+          different names for each script. If it is not defined and
+          <literal>location</literal> is defined, it will use filename from
```

Comment:
> it will use filename from

What will use (the) filename? Filename of what? What is (the) "location path"?

Reply:
@lproven let me show you example - 

You have script test.sh at ftp://neser-vr.suse.cz/test.sh
in autoyast profile you do *not* specify filename and has only <location>ftp://neser-vr.suse.cz/test.sh</location> . So autoyast use as filename "test.sh" ( basically path from parsed URL and apply on it basename so even when it is scripts/test.sh it will use just test.sh )

Reply:
OK. 

What I was asking was what "it" meant in all these contexts. If "it" usually means "autoyast" then that is how I will edit it to read.

Repetition is more permissible in documentation than in normal writing. :-)